src/script/auth/page/OAuthPermissions.tsx

```diff
@@ -0,0 +1,285 @@
+        } else {
+          const teamImageBlob = await assetRepository.load(teamIcon);
+          setTeamImage(teamImageBlob && (await loadDataUrl(teamImageBlob)));
+        }
+      }
+      setOAuthApp(!!oauthParams.client_id ? await getOAuthApp(oauthParams.client_id) : null);
+    };
+    getUserData().catch(error => {
+      console.error(error);
+    });
+  }, [
+    assetRepository,
+    getOAuthApp,
+    getSelf,
+    getTeam,
+    oauthParams.client_id,
+    selfTeamId,
+    selfUser.qualified_id?.domain,
+  ]);
+
+  return (
+    <Page>
+      <ContainerXS
+        centerText
+        verticalCenter
+        style={{width: '100%', display: 'flex', flexDirection: 'column', alignItems: 'center'}}
+      >
+        {!oAuthApp ? (
+          <Icon.Loading width="36" height="36" css={{path: {fill: COLOR_V2.BLUE_DARK_500}}} />
+        ) : (
+          <>
+            <H2 style={{fontWeight: 500, lineHeight: '28.64px', fontSize: '24px'}}>{_(oauthStrings.headline)}</H2>
+            {typeof teamImage === 'string' && (
+              <img
+                src={teamImage}
+                style={{
+                  width: '22px',
+                  height: '22px',
+                  borderRadius: '6px',
+                  border: 'black 1px solid',
+                  padding: '2px',
+                  margin: '15px',
+                }}
+                alt="teamIcon"
+              />
+            )}
+            <Text style={{marginBottom: '8px'}}>{selfUser.email}</Text>
+            <Link
+              style={{marginBottom: '32px'}}
+              onClick={doLogout}
+              data-uie-name="go-logout"
+              variant={LinkVariant.PRIMARY}
+              color={COLOR_V2.BLUE}
+            >
+              {_(oauthStrings.logout)}
+            </Link>
+
+            <Text style={{marginBottom: '24px'}}>{_(oauthStrings.subhead, {app: oAuthApp?.application_name})}</Text>
+
+            {oauthParams.scope.length > 1 && (
+              <Box
+                style={{
+                  marginBottom: '24px',
+                  background: COLOR_V2.GRAY_20,
+                  borderColor: COLOR_V2.GRAY_20,
+                  padding: '8px',
+                }}
+              >
+                <ul
+                  style={{
+                    marginTop: 0,
+                    paddingInlineStart: '20px',
+                    fontSize: '12px',
+                  }}
+                >
+                  {oauthScope.map((scope, index) => (
+                    <li key={index} style={{textAlign: 'start'}}>
+                      <Text>{_(oauthStrings[scope])}</Text>
+                    </li>
+                  ))}
+                </ul>
+                <Text
+                  data-uie-name="oauth-learn-more"
+                  style={{
+                    fontSize: '12px',
+                    lineHeight: '16px',
+                    display: 'block',
+                  }}
```

Comment:
Let's extract all those styles to a `OAuthPermissions.styles.ts` and put it to the `css={}` attributes (not `style`)

Reply:
done